src/Umbraco.Core/Services/DataTypeService.cs

```diff
@@ -462,7 +462,8 @@ private void ConvertMissingEditorsOfDataTypesToLabels(IEnumerable<IDataType> dat
 
                     copy.Name += " (copy)"; // might not be unique
                     copy.ParentId = containerId;
-                    _dataTypeRepository.Save(copy);
+
+                    Save(copy);
```

Comment:
This resets the `CreatorId` to the super user ID (-1), so we should at least keep the existing one:

```suggestion
                    Save(copy, copy.CreatorId);
```

This method should preferably also get the same `userId` parameter, but that might be a breaking change (I'm not sure when this was added to the interface)...

Reply:
The Copy interface will be added in 10.3, so does it then still count as a breaking change if it hasn't been released yet?